docker-compose.yaml

```diff
@@ -98,4 +97,22 @@ services:
               count: 1
               capabilities: [gpu]
     profiles:
-      - ollama-cuda
\ No newline at end of file
+      - ollama-cuda
+
+    # fireworks service
+  private-gpt-fireworks:
+    build:
+      context: .
+      dockerfile: Dockerfile.fireworks
+    volumes:
+      - ./local_data/:/home/worker/app/local_data
+    ports:
+      - "3001:8080"
+    environment:
+      PORT: 8080
+      PGPT_PROFILES: fireworks
+      FIREWORKS_API_KEY: ${FIREWORKS_API_KEY}
+    env_file:
+      - .env
+    profiles:
+      - fireworks
```

Comment:
Same as Dockerfile

Reply:
here i have used how other docker files and docker-compose are written
followed those template so that it won't be outsider :)

Reply:
I would say the same. It is not the core of PGPT, therefore, we should not give this kind of support. Our goal is to give a 100% private solution, in this area, our two main providers are Ollama and Llama-CPP. Of course, this PR gives more value to other people with the same problems, but it doesn't make sense to maintain on docker-compose and Dockerfile.